### DISCUSSION TITLE
devise-omniauth-github

### DISCUSSION THREAD

**Discussion by giscus** [yshmarov/yshmarov.github.io#12] [General]
# devise-omniauth-github

If we are using Devise for user authentication, it’s easy to add omniauth authentication (via a social provider). Let’s use omniauth github gem to allow “Login via Github”.

https://blog.corsego.com/devise-omniauth-github

**Comment by secretpray** [comment]
Дякую!
Гарний матеріал.
Завжди буде в нагоді!
